en/docs/tutorials/create-rest-api-component.md

```diff
@@ -0,0 +1,299 @@
+# Create a REST API
+
+A REST API is an Application Programming Interface that conforms to the constraints of REST architectural styles and principles. REST is an [architectural style](https://www.ics.uci.edu/~fielding/pubs/dissertation/rest_arch_style.htm) introduced by Dr. Roy Fielding in his doctoral dissertation in the year 2000. It was initially designed as a guide for the development of the web. Today, REST is being widely adopted for the creation of high-performance, stateless and reliable APIs that are used by numerous applications.
+
```

Comment:
We need to mention what we are covering in the tutorial as steps.

Reply:
Added some text for this.